lua/nvim-web-devicons/icons-default.lua

```diff
@@ -524,6 +770,18 @@ local icons_by_file_extension = {
     cterm_color = "185",
     name = "BibTeX",
   },
+  ["bin"] = {
```

Comment:
Let's not do this one please - ELF is architecture and operating system specific.

Reply:
ok, I saw there was a typo you mean to fix it because it says Elf instead of bin? Or totally remove it?

Reply:
Bin could be anything at all, not just an executable.

We could use nf-cod-file_binary

Reply:
I'm already using `nf-cod-file_binary` for bin, same as in elf

Reply:
I see. Please accept my apologies.

Reply:
Sure, don't worry